selfdrive/car/honda/fingerprints.py

```diff
@@ -1433,4 +1433,9 @@
       b'37805-64S-AA10\x00\x00',
     ],
   },
+  CAR.CRV_HYBRID_6G: {
+    (Ecu.fwdRadar, 0x18dab0f1, None): [
+      b'8S302-3D4-A050\x00\x00',
+    ],
+  },
```

Comment:
Any idea why there's only one ECU? Can we do an ECU scan on this car to search for other addresses?

Reply:
the first route I got only had this version. I have newer routes that have the fwdcamera, but nothing else